tests/test_dijkstra.py

```diff
@@ -39,6 +39,240 @@ def test_dijkstra(self):
         expected = {4: 23.0}
         self.assertEqual(expected, path)
 
+    def test_dijkstra_path(self):
+        g = retworkx.PyDiGraph()
```

Comment:
If this is the same graph for every test, could it be moved into a `setUp`? Or is there a diff that I'm not catching?

Reply:
Done in: https://github.com/Qiskit/retworkx/pull/162/commits/06731356ccbfd37a408b4c3bbe9dfc7fe3e7bbda